app/models/movie.rb

```diff
@@ -1,3 +1,13 @@
 class Movie < ApplicationRecord
   belongs_to :studio
+  has_many :movie_actors
+  has_many :actors, through: :movie_actors
+
+  def actors_youngest_to_oldest
+    actors.order(:age)
+  end
+
+  def actors_average_age
+    actors.average(:age)
+  end
```

Comment:
You commented on your submission that you weren't sure where to put these methods and this looks good to me. Since we care about the actors of a specific movie, creating instance methods on movies is logical. If we put them on `Actor` then they would only affect individual instances of actors (and we're sorting/averaging multiple) or ALL actors if it were a class method. But we don't want ALL actors. We just want all associated with one instance of a movie. Hence, your choice was sound to put these methods here.